src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx

```diff
@@ -205,9 +208,74 @@ export class ArtworkSidebarCommercialContainer extends React.Component<
     })
   }
 
+  handleCreateOfferOrder = () => {
+    this.setState({ isCommittingCreateOfferOrderMutation: true }, () => {
+      if (this.props.relay && this.props.relay.environment) {
+        commitMutation<ArtworkSidebarCommercialOfferOrderMutation>(
+          this.props.relay.environment,
+          {
+            mutation: graphql`
+              mutation ArtworkSidebarCommercialOfferOrderMutation(
+                $input: CreateOfferOrderWithArtworkInput!
+              ) {
+                ecommerceCreateOfferOrderWithArtwork(input: $input) {
+                  orderOrError {
+                    ... on OrderWithMutationSuccess {
+                      __typename
+                      order {
+                        id
+                        mode
+                      }
+                    }
+                    ... on OrderWithMutationFailure {
+                      error {
+                        type
+                        code
+                        data
+                      }
+                    }
+                  }
+                }
+              }
+            `,
+            variables: {
+              input: {
+                artworkId: this.props.artwork.id,
+                editionSetId:
+                  this.state.selectedEditionSet &&
+                  this.state.selectedEditionSet.id,
+              },
+            },
+            onCompleted: data => {
+              this.setState({ isCommittingCreateOrderMutation: false })
+              const {
+                ecommerceCreateOfferOrderWithArtwork: { orderOrError },
+              } = data
+              if (orderOrError.error) {
+                this.onMutationError(
+                  new ErrorWithMetadata(
+                    orderOrError.error.code,
+                    orderOrError.error
+                  )
+                )
+              } else {
+                window.location.assign(`/orders/${orderOrError.order.id}/offer`)
+              }
+            },
+            onError: this.onMutationError.bind(this),
+          }
+        )
+      }
+    })
+  }
```

Comment:
Looks great 👍 would you mind digging into the [order app](https://github.com/artsy/reaction/blob/master/src/Apps/Order/Routes/__tests__/) to find an example of a mutation test and add a bit of coverage?